### ISSUE TITLE
TestSwingingDoorTrendingFilterProcessor

### ISSUE THREAD

**Issue by bossenti** [apache/streampipes#2773]

**Comment by pambrocio** [comment]
successful build

```
[INFO] 
[INFO] Apache StreamPipes ................................. SUCCESS [  1.796 s]
[INFO] streampipes-archetype-extensions-jvm ............... SUCCESS [  0.877 s]
[INFO] streampipes-archetype-pe-sinks-flink ............... SUCCESS [  0.073 s]
[INFO] streampipes-archetype-pe-processors-flink .......... SUCCESS [  0.066 s]
[INFO] StreamPipes Commons ................................ SUCCESS [  2.654 s]
[INFO] streampipes-model-shared ........................... SUCCESS [  0.946 s]
[INFO] streampipes-vocabulary ............................. SUCCESS [  1.513 s]
[INFO] streampipes-model .................................. SUCCESS [  4.557 s]
[INFO] streampipes-dataformat ............................. SUCCESS [  0.896 s]
[INFO] streampipes-messaging .............................. SUCCESS [  0.934 s]
[INFO] streampipes-client-api ............................. SUCCESS [  1.122 s]
[INFO] streampipes-dataformat-json ........................ SUCCESS [  0.865 s]
[INFO] streampipes-dataformat-cbor ........................ SUCCESS [  0.947 s]
[INFO] streampipes-dataformat-fst ......................... SUCCESS [  0.867 s]
[INFO] streampipes-security-jwt ........................... SUCCESS [  0.954 s]
[INFO] streampipes-model-client ........................... SUCCESS [  0.627 s]
[INFO] streampipes-extensions-api ......................... SUCCESS [  1.408 s]
[INFO] streampipes-test-utils ............................. SUCCESS [  1.081 s]
[INFO] streampipes-serializers-json ....................... SUCCESS [  1.202 s]
[INFO] streampipes-client ................................. SUCCESS [  1.103 s]
[INFO] streampipes-measurement-units ...................... SUCCESS [  1.685 s]
[INFO] streampipes-sdk .................................... SUCCESS [  3.366 s]
[INFO] streampipes-connect-shared ......................... SUCCESS [  2.084 s]
[INFO] streampipes-service-discovery-api .................. SUCCESS [  0.885 s]
[INFO] streampipes-dataformat-smile ....................... SUCCESS [  0.848 s]
[INFO] streampipes-storage-api ............................ SUCCESS [  1.048 s]
[INFO] streampipes-storage-couchdb ........................ SUCCESS [  2.227 s]
[INFO] streampipes-storage-management ..................... SUCCESS [  0.837 s]
[INFO] streampipes-user-management ........................ SUCCESS [  1.594 s]
[INFO] streampipes-mail ................................... SUCCESS [  1.234 s]
[INFO] streampipes-resource-management .................... SUCCESS [  2.325 s]
[INFO] streampipes-service-discovery ...................... SUCCESS [  0.914 s]
[INFO] streampipes-pipeline-management .................... SUCCESS [  3.371 s]
[INFO] streampipes-connect-management ..................... SUCCESS [  2.571 s]
[INFO] streampipes-extensions-management .................. SUCCESS [  2.657 s]
[INFO] streampipes-data-explorer-commons .................. SUCCESS [  2.522 s]
[INFO] streampipes-data-explorer .......................... SUCCESS [  2.002 s]
[INFO] streampipes-data-export ............................ SUCCESS [  1.405 s]
[INFO] streampipes-service-base ........................... SUCCESS [  1.009 s]
[INFO] streampipes-rest-shared ............................ SUCCESS [  1.002 s]
[INFO] streampipes-rest-extensions ........................ SUCCESS [  1.983 s]
[INFO] streampipes-service-extensions ..................... SUCCESS [  1.372 s]
[INFO] Apache StreamPipes Maven Plugin .................... SUCCESS [  3.837 s]
[INFO] streampipes-extensions ............................. SUCCESS [  0.007 s]
[INFO] streampipes-connectors-mqtt ........................ SUCCESS [  1.127 s]
[INFO] streampipes-pipeline-elements-shared ............... SUCCESS [  0.751 s]
[INFO] streampipes-connect-adapters ....................... SUCCESS [  0.770 s]
[INFO] streampipes-connect-adapters-iiot .................. SUCCESS [  2.055 s]
[INFO] streampipes-wrapper ................................ SUCCESS [  1.275 s]
[INFO] StreamPipes Wrapper for Standalone Pipeline Element Implementations SUCCESS [  1.355 s]
[INFO] streampipes-connectors-influx ...................... SUCCESS [  1.242 s]
[INFO] streampipes-messaging-kafka ........................ SUCCESS [  1.046 s]
[INFO] streampipes-connectors-kafka ....................... SUCCESS [  1.174 s]
[INFO] streampipes-messaging-nats ......................... SUCCESS [  0.859 s]
[INFO] streampipes-connectors-nats ........................ SUCCESS [  1.014 s]
[INFO] streampipes-connectors-opcua ....................... SUCCESS [  2.621 s]
[INFO] streampipes-connectors-pulsar ...................... SUCCESS [  2.627 s]
[INFO] streampipes-connectors-rocketmq .................... SUCCESS [  2.557 s]
[INFO] streampipes-connectors-tubemq ...................... SUCCESS [  1.092 s]
[INFO] streampipes-processors-filters-jvm ................. SUCCESS [  3.823 s]
[INFO] streampipes-connectors-plc ......................... SUCCESS [  1.805 s]
[INFO] streampipes-messaging-mqtt ......................... SUCCESS [  0.857 s]
[INFO] streampipes-messaging-pulsar ....................... SUCCESS [  1.008 s]
[INFO] streampipes-wrapper-siddhi ......................... SUCCESS [  1.831 s]
[INFO] streampipes-processors-filters-siddhi .............. SUCCESS [  1.802 s]
[INFO] streampipes-processors-transformation-jvm .......... SUCCESS [  4.217 s]
[INFO] streampipes-messaging-jms .......................... SUCCESS [  0.900 s]
[INFO] streampipes-sinks-brokers-jvm ...................... SUCCESS [  1.297 s]
[INFO] streampipes-sinks-databases-jvm .................... SUCCESS [  1.589 s]
[INFO] streampipes-sinks-internal-jvm ..................... SUCCESS [  2.138 s]
[INFO] streampipes-sinks-notifications-jvm ................ SUCCESS [  2.596 s]
[INFO] streampipes-processors-enricher-jvm ................ SUCCESS [  1.632 s]
[INFO] streampipes-processors-change-detection-jvm ........ SUCCESS [  1.110 s]
[INFO] streampipes-extensions-all-iiot .................... SUCCESS [  2.004 s]
[INFO] streampipes-processors-image-processing-jvm ........ SUCCESS [  1.362 s]
[INFO] streampipes-processors-text-mining-jvm ............. SUCCESS [  1.167 s]
[INFO] streampipes-processors-geo-jvm ..................... SUCCESS [  1.747 s]
[INFO] streampipes-extensions-all-jvm ..................... SUCCESS [  3.429 s]
[INFO] streampipes-extensions-iiot-minimal ................ SUCCESS [  1.656 s]
[INFO] StreamPipes Integration Tests ...................... SUCCESS [ 26.262 s]
[INFO] streampipes-rest-core-base ......................... SUCCESS [  1.111 s]
[INFO] StreamPipes REST API ............................... SUCCESS [  2.283 s]
[INFO] streampipes-platform-services ...................... SUCCESS [  1.267 s]
[INFO] streampipes-sdk-bundle ............................. SUCCESS [  0.185 s]
[INFO] StreamPipes Core ................................... SUCCESS [  2.659 s]
[INFO] streampipes-service-core-minimal ................... SUCCESS [  1.144 s]
[INFO] streampipes-wrapper-distributed .................... SUCCESS [  0.981 s]
[INFO] StreamPipes Wrapper for Apache Flink ............... SUCCESS [  1.675 s]
[INFO] streampipes-wrapper-kafka-streams .................. SUCCESS [  1.116 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:45 min
[INFO] Finished at: 2024-04-25T11:09:59+08:00
[INFO] ------------------------------------------------------------------------
paulo@PRED:~/ASF/streampipes$ git status
On branch 2773-fix-TestSwingingDoorTrendingFilterProcessor
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   streampipes-extensions/streampipes-processors-filters-jvm/src/test/java/org/apache/streampipes/processors/filters/jvm/processor/sdt/TestSwingingDoorTrendingFilterProcessor.java

no changes added to commit (use "git add" and/or "git commit -a")
paulo@PRED:~/ASF/streampipes$ 

```

**Comment by pambrocio** [comment]
PR https://github.com/apache/streampipes/pull/2787

**Comment by tenthe** [comment]
Fixed in PR #2375.
